preprocessing/train_val_split.ipynb

```diff
@@ -0,0 +1,83 @@
+{
+ "cells": [
```

Comment:
In general I'm a bit confused about the two files - what should be run to get a validation set? Is the notebook just for debugging? Or is it the other way around?